cmd/kube-spawn/create.go

```diff
@@ -88,7 +91,10 @@ func doCreate() {
 		KubernetesVersion:   viper.GetString("kubernetes-version"),
 		KubernetesSourceDir: viper.GetString("kubernetes-source-dir"),
 		CNIPluginDir:        viper.GetString("cni-plugin-dir"),
+                CNIModel:            viper.GetString("cni-model"),
```

Comment:
Let's use tab instead of space, following the Golang's standard coding style.
Running gofmt will help you fix such issues automatically.